R/safetyGraphicsApp.R

```diff
@@ -51,28 +51,34 @@ safetyGraphicsApp <- function(
     homeTabPath=homeTabPath
   )
 
-  app <- shinyApp(
-    ui =  safetyGraphicsUI(
+    ui <- safetyGraphicsUI(
         "sg",
         config$meta,
         config$domainData,
         config$mapping,
         config$standards,
         config
-    ),
-    server = function(input, output, session) {
-      callModule(
-        safetyGraphicsServer,
-        "sg",
-        config$meta, 
-        config$mapping, 
-        config$domainData, 
-        config$charts, 
-        config$filterDomain,
-        config
-      )
+    )
+
+    server <- function(input, output, session) {
+        module_outputs <- callModule(
```

Comment:
Don't think we want to do this.

Reply:
If we have the server function return something, I'd think it should be a larger config including `meta`, `mapping` etc. For now, I don't think the overall server function is well designed to be used as part of a larger app, so I'd lean towards not implementing this for now.

Reply:
Removed.